### ISSUE TITLE
“Lombok Dependency is missing” warning when the dependency is present

### ISSUE THREAD

**Issue by mirabilos** [mplushnikov/lombok-intellij-plugin#820]
### Short description

When a Lombok dependency is present, the warning is still there.

### Expected behavior

The warning to not be displayed. (Or even consider [whether the warning is useful at all](https://github.com/mplushnikov/lombok-intellij-plugin/issues/819).)

### Version information

 * **IDEA Version:** 
    IntelliJ IDEA 2020.1.2 (Ultimate Edition)
    Build #IU-201.7846.76, built on June 1, 2020
    Licensed to […]
    Runtime version: 11.0.7+10-post-Debian-3 amd64
    VM: OpenJDK 64-Bit Server VM by Debian
    Linux 5.6.0-2-amd64
    GC: G1 Young Generation, G1 Old Generation
    Memory: 2048M
    Cores: 2
    Non-Bundled Plugins: Switch Structure, Base64 for IDEA and Storm, Realigner, AlphaTab, AutoConstructor, CheckStyle-IDEA, GenerateSerialVersionUID, IntelliJDeodorant, Lombook Plugin, Time Convertor Plugin, Wrap in StringBuilder, com.dmitz.intellij.plugin.websocket.client, com.piegoesmoo.escape, com.viartemev.requestmapper, jd-intellij, me.guichaguri.additionaltools, MavenRunHelper, com.mycila.intellij.plugin.gmavenplus, com.github.platan.gradle-dependencies-formatter, com.jetbrains.plugins.ini4idea, com.ultrahob.zerolength.plugin2, mobi.hsz.idea.gitignore, org.jetbrains.plugins.vue, tv.twelvetone.intellij.plugins.intellivue, some.awesome
    Current Desktop: Undefined

 * **JDK Version:** openjdk-11-jdk 11.0.7+10-3
 * **OS Type & Version:** Debian GNU/Linux sid/amd64
 * **Lombok Plugin Version:** 0.30-2020.1
 * **Lombok Dependency Version:** 

```
    <properties>
                <lombok.version>1.18.12</lombok.version>
    </properties>
        <dependencyManagement>
                <dependencies>
                        <dependency>
                                <groupId>org.projectlombok</groupId>
                                <artifactId>lombok</artifactId>
                                <version>${lombok.version}</version>
                                <scope>provided</scope>
                        </dependency>
                </dependencies>
        </dependencyManagement>
        <dependencies>
                <dependency>
                        <groupId>org.projectlombok</groupId>
                        <artifactId>lombok</artifactId>
                        <scope>provided</scope>
                </dependency>
        </dependencies>
```

### Steps to reproduce

Load a pre-existing Maven project that uses Lombok with IntelliJ after upgrading it and the Lombok plugin to 2020. (This did not warn in 2019.)

### Sample project

I can do that but only if really needed.

**Comment by fantaseour** [comment]
Was hit with this issue today. My lombok version was 1.18.8. 

First -- change lombok version to 1.18.12 solve the problem. 
Second: I've cleared lombok fom my .m2 folder (org.projectlombok), then switch back to my 1.18.8 and it works...

Hope this helps

**Comment by hcguersoy** [comment]
> Hope this helps

Unfortunately mot in my case :-(

**Comment by mirabilos** [comment]
This happens on https://github.com/tarent/VerA.web for example.

**Comment by mplushnikov** [comment]
> This happens on https://github.com/tarent/VerA.web for example.

Cannot reproduce it now, did you already changed something in this project?

**Comment by mirabilos** [comment]
Michail Plushnikov dixit:

>> This happens on https://github.com/tarent/VerA.web for example.
>
>Cannot reproduce it now, did you already changed something in this project?

I cannot reproduce it for that project any more either. I also found
the setting to disable the warning and did that for new projects.

But I just upgraded the Lombok plugin earlier today in another project
and *did* get this warning, even though it uses Lombok, so 0.31 also
fails detecting Lombok sometimes. Now all other projects use Maven, but
the one from earlier today is an Android project… so no discernible pattern.

bye,
//mirabilos
-- 
<ch> you introduced a merge commit        │<mika> % g rebase -i HEAD^^
<mika> sorry, no idea and rebasing just fscked │<mika> Segmentation
<ch> should have cloned into a clean repo      │  fault (core dumped)
<ch> if I rebase that now, it's really ugh     │<mika:#grml> wuahhhhhh
